app/Libraries/TeamSpeak.php

```diff
@@ -275,26 +275,35 @@ public static function checkMemberMandatoryNotifications(TeamSpeak3_Node_Client
      */
     public static function checkClientNickname(TeamSpeak3_Node_Client $client, Account $member)
     {
-        if (!$member->isValidDisplayName($client['client_nickname'])) {
+        if (!$member->isValidDisplayName($client['client_nickname']) && !$member->isDuplicateDisplayName($client['client_nickname'])) {
```

Comment:
Can you drop this onto two lines to make it a bit more readable?

Reply:
I personally don't think its too long / complicated, and feel if it was split into two lines it would just leave redundant braces and clutter it up? Its probably on the shorted end of some of our if's :P